.travis.yml

```
@@ -20,12 +20,14 @@ addons:
 before_install:
     - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH"
     - export ENSEMBL_BRANCH=master
-    - if [[ $TRAVIS_BRANCH =~ ^release\/[0-9]+$ ]]; then export ENSEMBL_BRANCH=$TRAVIS_BRANCH; fi
+    - export METADATA_BRANCH=main
+    - if [[ $TRAVIS_BRANCH =~ ^release\/[0-9]+$ ]]; then export ENSEMBL_BRANCH=$TRAVIS_BRANCH; export METADATA_BRANCH=$TRAVIS_BRANCH; fi
     - echo "ENSEMBL_BRANCH=$ENSEMBL_BRANCH"
+    - echo "METADATA_BRANCH=$METADATA_BRANCH"
     - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl.git
     - git clone --branch $ENSEMBL_BRANCH --depth 1 https://github.com/Ensembl/ensembl-funcgen.git
     - git clone --branch version/2.5 --depth 1 https://github.com/Ensembl/ensembl-hive.git
```

```suggestion
    - git clone --branch version/2.6 --depth 1 https://github.com/Ensembl/ensembl-hive.git
```
I don't think it'll make any difference, but might as well bump this while we're here.